### PR TITLE
DDR: Avoid numerous warnings due to empty 'stub' files

### PR DESCRIPTION
**Pull request by keithc-ca**
on AIX warnings appear like this:
>  "j9pool/pool.stub.c", line 1: 1506-229 (W) File is empty.

on z/OS warnings appear like this:
>  WARNING CCN3229 j9pool/pool.stub.c:1     File is empty.

### CODE REVIEW

**Review by dsouzai — APPROVED** [review]

**Comment by dsouzai** [comment]
jenkins build all

**Comment by dsouzai** [comment]
jenkins build zos

**Comment by dsouzai** [comment]
z/OS builds are broken until [1] is resolved.

[1] https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/999

**Comment by dsouzai** [comment]
FWIW (for posterity), I'm holding off merging this only because the change was added specifically for AIX and z/OS, so I'd like to build z/OS before merging.

**Comment by dsouzai** [comment]
As for the riscv failure, seems to be an existing issue as it was seen in https://github.com/eclipse/omr/pull/6316#issuecomment-1027198038 too.

**Comment by dsouzai** [comment]
jenkins build zos

**Comment by keithc-ca** [comment]
jenkins build zos

**Comment by dsouzai** [comment]
No "File is empty" warnings seen on z/OS or AIX; merging.

**Pull request merged**
